docs/content/v0.20/examples.md

```diff
@@ -0,0 +1,588 @@
+```
+Finally, scale the deployment from 1 to 2 replicas in wds1:
+
+```shell
+kubectl --context wds1 scale deployment nginx-singleton-deployment --replicas=2
+```
+
+and verify that replicas has been updated in cluster1 and wds1:
+
+```shell
+kubectl --context cluster1 get deployment nginx-singleton-deployment
+kubectl --context wds1 get deployment nginx-singleton-deployment
+```
+
+## Scenario 5 - Resiliency testing
+
+This is a test that you can do after finishing Scenario 1.
+
+TODO: rewrite this so that it makes sense after Scenario 4.
+
+Bring down the control plane: stop and restart wds1 and imbs1 API servers, 
+KubeFlex and KubeStellar controllers:
+
+First stop all:
+
+```shell
+kubectl --context kind-kubeflex scale deployment -n wds1-system kube-apiserver --replicas=0
+kubectl --context kind-kubeflex scale statefulset -n imbs1-system vcluster --replicas=0
+kubectl --context kind-kubeflex scale deployment -n kubeflex-system kubeflex-controller-manager --replicas=0
+kubectl --context kind-kubeflex scale deployment -n wds1-system kubestellar-controller-manager --replicas=0
+```
+
+Then restart all:
+
+```shell
+kubectl --context kind-kubeflex scale deployment -n wds1-system kube-apiserver --replicas=1
+kubectl --context kind-kubeflex scale statefulset -n imbs1-system vcluster --replicas=1
+kubectl --context kind-kubeflex scale deployment -n kubeflex-system kubeflex-controller-manager --replicas=1
+kubectl --context kind-kubeflex scale deployment -n wds1-system kubestellar-controller-manager --replicas=1
+```
+
+Wait for about a minute for all pods to restart, then apply a new placement:
+
+```shell
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: edge.kubestellar.io/v1alpha1
+kind: Placement
+metadata:
+  name: nginx-res-placement
+spec:
+  clusterSelectors:
+  - matchLabels: {"location-group":"edge"}
+  downsync:
+  - objectSelectors:
+    - matchLabels: {"app.kubernetes.io/name":"nginx-res"}
+EOF
+```
+
+and a new workload:
+
+```shell
+kubectl --context wds1 apply -f - <<EOF
+apiVersion: v1
+kind: Namespace
+metadata:
+  labels:
+    app.kubernetes.io/name: nginx-res
+  name: nginx-res
+---
+apiVersion: apps/v1
+kind: Deployment
+metadata:
+  name: nginx-res-deployment
+  namespace: nginx-res
+  labels:
+    app.kubernetes.io/name: nginx-res
+spec:
+  replicas: 1
+  selector:
+    matchLabels:
+      app: nginx-res
+  template:
+    metadata:
+      labels:
+        app: nginx-res
+    spec:
+      containers:
+      - name: nginx-res
+        image: public.ecr.aws/nginx/nginx:latest 
+        ports:
+        - containerPort: 80
+EOF
+```
+
+Verify that deployment has been created in both clusters
+
+```shell
+kubectl --context cluster1 get deployments -n nginx-res
+kubectl --context cluster2 get deployments -n nginx-res
+```
+
+## Scenario 6 - multi-cluster workload deployment of app with ServiceAccount with ArgoCD 
+
+This scenario is something you can do after the [common setup](#common-setup).
```

Comment:
Looks like when Scenario 3 is deleted, the instructions for Argo CD installation, i.e. the reference to thirdparties.md, which was at the beginning of the deleleted Scenario 3, also gets deleted. We need to get that reference back, right?

Reply:
Good catch. Restored.